src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -2730,6 +2730,15 @@ public void TestInCollectionInitializers2()
 @"using System ; class C { void M() { var x = new System.Collections.Generic.Dictionary<int, bool> { { 1, T() } }; } private bool T() { throw new NotImplementedException(); } } ");
         }
 
+        [WorkItem(774321)]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
+        public void TestEquivalenceKey()
+        {
+            TestEquivalenceKeyWorker(
```

Comment:
@dpoeschl The name of the helper is a bit weird - It can seem as though the test is testing some feature named 'equivalencekeyworker' :) I would rather change the name of the test to TestGenerateMethodEquivalenceKey or some such and keep the name of the helper same as before (TestEquivalenceKey)...